homeassistant/components/derivative/config_flow.py

```diff
@@ -51,7 +50,7 @@
             ),
         ),
         vol.Required(CONF_TIME_WINDOW): selector.DurationSelector(),
-        vol.Required(CONF_UNIT_PREFIX, default="none"): selector.SelectSelector(
+        vol.Optional(CONF_UNIT_PREFIX, default=None): selector.SelectSelector(
```

Comment:
When the item has a default, `Optional` and `Required` behave exactly the same from a `voluptuous` point of view. Does frontend make a difference between the two, i.e. the clearing the select selector only works if the item is `Optional`?

Reply:
> Does frontend make a difference between the two, i.e. the clearing the select selector only works if the item is `Optional`?

Yes clearing will only showing up on optional selectors.